core/domain/opportunity_services.py

```diff
@@ -91,7 +91,45 @@ def get_exploration_opportunity_summary_from_model(model):
         model.story_title, model.chapter_title, model.content_count,
         new_incomplete_translation_language_codes, model.translation_counts,
         model.language_codes_needing_voice_artists,
-        model.language_codes_with_assigned_voice_artists)
+        model.language_codes_with_assigned_voice_artists,
+        {})
+
+
+def get_exploration_opportunity_summary_with_in_review_translations_from_model(
```

Comment:
You might do this if you want
```suggestion
def get_exp_opportunity_summary_with_in_review_translations_from_model(
```

Reply:
Done. Thanks!